src/troute-network/troute/NHDNetwork_v4.py

```diff
@@ -0,0 +1,318 @@
+from  .AbstractNetwork import AbstractNetwork
```

Comment:
This file should simply replace the existing NHDNetwork.py file.  This way the changes from one version to the next are marked in the version control.  A well crafted commit message for this single change will make going back into the history and finding the old version straight forward, and we don't have multiple files to maintain.